runtime/src/runtime.js

```diff
@@ -49,27 +45,34 @@ const tracingImports = TRACE_MEMORY ? {
   decRefIgnoreZeros: managedMemory.decRefIgnoreZeros.bind(managedMemory)
 };
 
+function debugPrint(v) {
+  console.log(`0x${v.toString(16)} (0b${v.toString(2)})`);
+  return v;
+}
+
 const importObj = {
   env: {
-    memory
+    memory,
+    // [TODO] Runner crashes without this. Why? Not referenced anywhere in our code
```

Comment:
Host needs to provide an `abort()` function when using some WASI calls. This should be getting supplied by wasmerjs, so we may not be providing the WASI imports properly somewhere.
```suggestion
```

Reply:
@ospencer Where are we providing those imports (or do you mean that we may have forgotten)? Maybe there's a rogue codepath which is missing them.